bundles/org.openhab.binding.powermax/README.md

```diff
@@ -132,7 +138,7 @@ Here is an example of command you can run: `smarthome:powermax powermax:serial:h
 ## Notes & Limitations
 
 * For Powerlink mode to work, the enrollment procedure has to be followed. If you don't enroll the Powerlink on the PowerMax the binding will operate in Standard mode, and if enrolled in Powerlink mode. On the newer software versions of the PowerMax the Powerlink enrollment is automatic, and the binding should only operate in 'Powerlink' mode (if enrollment is successful).
-* In Powerlink mode, the binding is downloading the panel setup at startup. When openHAB is starting, unfortunately this download is often failing on a RPI for an unclear reason (maybe too many things running at the same time). A retry mechanism is implemented in the binding to retry until 3 times with a delay of 1 minute between each try. My experience is that the download finally succeeds. In case it fails after all the retries, you still have the option to later trigger the download either by using the channel `download_setup` or the appropriate console command.
+* In Powerlink mode, the binding is downloading the panel setup at startup. When openHAB is starting, unfortunately this download is often failing on a Raspberry Pi for an unclear reason (maybe too many things running at the same time). A retry mechanism is implemented in the binding to retry until 3 times with a delay of 1 minute between each try. My experience is that the download finally succeeds. In case it fails after all the retries, you still have the option to later trigger the download either by using the channel `download_setup` or the appropriate console command.
```

Comment:
One line per sentence is possible in lists too.
Rendering is done correctly. :)

Reply:
That's a good one! I've updated that as well. It would be nice if there was a formatter to automate this. :-)